volcli/volcli.go

```diff
@@ -133,7 +134,11 @@ func tenantUpload(ctx *cli.Context) (bool, error) {
 		return false, err
 	}
 
-	tenant := &config.TenantConfig{}
+	tenant := &config.TenantConfig{
```

Comment:
yeah, I agree the repetition can be avoided by using a function for default config.

I am wondering if having `config.NewDefaultTenantConfig()` method will be better which will return an empty config with `backend` set to `ceph`. Does it sound right?

Later we can may be add other things to this method as well like I saw in volcli tests we by-default add a `"ext4": "mkfs.ext4 -m0 %"` filesystem to the config options. But I might be wrong about it's purpose.